### DISCUSSION TITLE
installation guide for mac

### DISCUSSION THREAD

**Discussion by ksdavidc** [input-leap/input-leap#1484] [Q&A]
It looks like there is not standalone version available yet. I am somewhat of a newbie and I would like ot try to install it on my macs but I am not savvy enough to understand how one does that. The "latest" release page is empty, and the repology doesn't mean anything to me. Could someone outline what is necessary to install on a mac. I assume I need c++? DO I need anything else? what are the steps. Consider me an "educated" amateur.

**Comment by bennypowers** [comment]
```
git clone git@github.com:input-leap/input-leap
cd input-leap
brew install gt5 openssl
./clean_build.sh
```

**Reply by Rich107** [reply]
why is this not in the readme am I looking in the wrong place for install instructions? 😕
